### PR TITLE
Fixing mac luancher bugs.

### PR DESCRIPTION
**Pull request by danteruiz**
Fixing launcher bugs - orgname case sensitive, password does not support special, crash when extracting interface zip file

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19803/HighFidelity-Beta-PR15690-f7333e9.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19803/HighFidelity-Quest-Beta-PR15690-f7333e9.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19803/HighFidelity-Beta-PR15690-f7333e9.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19803/HighFidelity-Beta-Interface-PR15690-f7333e9.dmg

File: launchers/darwin/src/CredentialsRequest.m [hunk 1]
```diff
@@ -7,7 +7,7 @@ @implementation CredentialsRequest
 - (void) confirmCredentials:(NSString*)username :(NSString*)password {
 
     NSLog(@"web request started");
-    NSString *post = [NSString stringWithFormat:@"grant_type=password&username=%@&password=%@&scope=owner", username, password];
+    NSString *post = [NSString stringWithFormat:@"grant_type=password&username=%@&password=%@&scope=owner", username, [password stringByAddingPercentEncodingWithAllowedCharacters:[NSCharacterSet symbolCharacterSet]]];
```
Comment (huffman) [inline launchers/darwin/src/CredentialsRequest.m]: I think we will want to encode both the username and password (and, in general, any externally inputted query components). Also, I believe the allowed character set should be `URLQueryAllowedCharacterSet` - thoughts?

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19803/HighFidelity-Beta-PR15690-f7333e9.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19803/HighFidelity-Beta-Interface-PR15690-f7333e9.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19812/HighFidelity-Beta-PR15690-5fe4f35.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19812/HighFidelity-Quest-Beta-PR15690-5fe4f35.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19812/HighFidelity-Beta-PR15690-5fe4f35.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19812/HighFidelity-Beta-Interface-PR15690-5fe4f35.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19812/HighFidelity-Beta-PR15690-5fe4f35.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19812/HighFidelity-Beta-Interface-PR15690-5fe4f35.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19827/HighFidelity-Beta-PR15690-5fe4f35.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19827/HighFidelity-Quest-Beta-PR15690-5fe4f35.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19827/HighFidelity-Beta-PR15690-5fe4f35.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19827/HighFidelity-Beta-Interface-PR15690-5fe4f35.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19827/HighFidelity-Beta-PR15690-5fe4f35.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19827/HighFidelity-Beta-Interface-PR15690-5fe4f35.dmg

**Pull request merged**